buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -4,10 +4,10 @@ pipeline {
 	parameters {
 		string (defaultValue: "https://github.com/AdoptOpenJDK/openjdk-tests.git", description: 'personal repo from AdoptOpenJDK/openjdk-tests', name: 'ADOPTOPENJDK_REPO')
 		string (defaultValue: "master", description: 'personal AdoptOpenJDK branch', name: 'ADOPTOPENJDK_BRANCH')
-		choice (choices: 'linux_x86-64\nlinux_x86-64_cmprssptrs\nmac_x86-64\nlinux_390-64\nlinux_390-64_cmprssptrs\nlinux_ppc-64_le\nlinux_ppc-64_cmprssptrs_le\nlinux_arm\nwin_x86\nwin_x86-64\nwin_x86-64_cmprssptrs\naix_ppc-64\naix_ppc-64_cmprssptrs\nzos_390-64_cmprssptrs', description: 'SPEC?', name: 'SPEC')
-		choice (choices: 'SE80\nSE90\nSE100', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
+		string (defaultValue: "linux_x86-64_cmprssptrs", description: 'Set SPEC to match SDK. e.g., <br/>linux_x86-64<br/>linux_x86-64_cmprssptrs<br/>mac_x86-64<br/>linux_390-64<br/>linux_390-64_cmprssptrs<br/>linux_ppc-64_le<br/>linux_ppc-64_cmprssptrs_le<br/>linux_arm<br/>win_x86<br/>win_x86-64<br/>win_x86-64_cmprssptrs<br/>aix_ppc-64<br/>aix_ppc-64_cmprssptrs<br/>zos_390-64_cmprssptrs', name: 'SPEC')
+		string (defaultValue: "SE80", description: 'Set JAVA_VERSION to match SDK. Format SEXXX. e.g., SE80 SE90 SE100 SE110', name: 'JAVA_VERSION')
 		string (defaultValue: 'openj9', description: 'JAVA_IMPL, e.g. hotspot, openj9, sap', name: 'JAVA_IMPL')
-		string (defaultValue: 'openjdk_regression', description: 'Equivalent to BUILD_LIST, specific test directory to compile, set blank for all projects to be compiled, e.g. openjdk_regression systemtest performance jck thirdparty_containers functional', name: 'TESTPROJECT')
+		string (defaultValue: 'functional', description: 'Specific test directory to compile, set blank for all projects to be compiled, e.g. openjdk_regression systemtest performance jck thirdparty_containers functional', name: 'BUILD_LIST')
```

Comment:
Probably other projects than functional as default would be more straightforward as there is no functional directory in adopt test repo?

Reply:
From my observation, it seems `functional` is more popular, so I made it as default value. If there are lots of people use `openjdk_regression`, I can change it back.